e2e/e2e_test.go

```diff
@@ -16,9 +16,10 @@ import (
 	"github.com/Azure/agentbaker/pkg/agent/datamodel"
 )
 
-// TODO 1: How to get the most accurate url links/image links for the currently hardcoded ones for eg CustomKubeBinaryURL, Pause Image etc
-// TODO 2: Update --rotate-certificate (true for TLS enabled, false otherwise, small nit)
-// TODO 3: Seperate out the certificate encode/decode
+/* TODO 1: How to get the most accurate url links/image links for the currently hardcoded ones for eg
```

Comment:
nit: single vs multiline comment syntax?

Reply:
I made the TODO 1: a multi line syntax because it goes over 111 characters. Are you asking that it be a single comment syntax or to put all 4 TODO steps into a multiline comment syntax?

Reply:
that's what I was thinking but not a big deal either way